shopfloor/models/shopfloor_menu.py

```diff
@@ -186,15 +193,22 @@ class ShopfloorMenu(models.Model):
     allow_alternative_destination_is_possible = fields.Boolean(
         compute="_compute_allow_alternative_destination_is_possible"
     )
-
     auto_post_line = fields.Boolean(
         string="Automatically post line",
         default=False,
-        help=AUTO_POST_LINE,
+        help=AUTO_POST_LINE_HELP,
     )
     auto_post_line_is_possible = fields.Boolean(
         compute="_compute_auto_post_line_is_possible"
     )
+    allow_alternative_destination_package = fields.Boolean(
+        string="Allow to transfer a destination package",
+        default=False,
```

Comment:
You need to add a migration script to enable this on existing menu using this scenario as you are adding a restriction that was not there before

Reply:
Added.